common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -111,6 +111,9 @@ public enum Feature {
         HOSTNAME_V1("Hostname Options V1", Type.DEFAULT),
         //HOSTNAME_V2("Hostname Options V2", Type.DEFAULT, 2),
 
+        PERSISTENT_USER_SESSIONS("Persistent online user sessions across restarts and upgrades", Type.PREVIEW),
```

Comment:
Is it really preview at this stage? Doesn't make more sense as a experimental feature?

Reply:
The main difference between experimental and preview is that preview is feature complete and that is provides some documentation. This is what we aim for in KC25. 

This PR already passes all the existing tests in the test suite, and serves as a drop-in-replacement for the existing user session handling. Given that, I think we're good to call it preview in the nightly build.  

Still, if this is the only remaining concern, I can change it experimental for now or in the next PR we're about to prepare. Let me know which way you decide.